embedded-files/fuseml-core-deployment.yaml

```diff
@@ -1,4 +1,52 @@
 ---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-core
+---
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: fuseml-core-tekton
+  namespace: fuseml-workloads
+rules:
+- apiGroups:
+  - tekton.dev
+  resources:
+  - pipelines
+  - pipelineruns
+  verbs:
+  - create
+  - get
+  - delete
+- apiGroups:
+  - triggers.tekton.dev
+  resources:
+  - eventlisteners
+  - triggerbindings
+  - triggertemplates
+  - triggers
```

Comment:
FuseML does not perform any operation on triggers

Reply:
done